lib/features/home/view_model/home_interaction_state.dart

```diff
@@ -18,3 +18,13 @@ class StartAddPropertyFlow extends HomeInteractionState {
   @override
   List<Object?> get props => [];
 }
+
+class StartOnTapBottomItems extends HomeInteractionState {
+  @override
+  List<Object?> get props => [];
+}
+
+class OnOpenBottomModal extends HomeInteractionState {
```

Comment:
this is a state, not an action.
```suggestion
class Open[Name]BottomSheetModal extends HomeInteractionState {
```
Should put the name of BottomSheet for clearer.